lib/services/terraform.js

```diff
@@ -299,6 +299,101 @@ class Terraform {
         return this.executeTerraform(name, 'tf-destroy', tfDestroySpawn);
     }
 
+    async status({ cloud, keys }) {
+        this.spinner = ora('Inspecting Orbs node').start();
+
+        const { name } = cloud;
+        this.setTargetPath(name);
+
+        if (fs.existsSync(path.join(this.targetPath, 'terraform.tfvars'))) {
+            this.spinner.succeed();
+            this.spinner.start(`Inspecting Terraform project internal state (reconciliation)..`);
+            await this.terraformRefresh({ name });
+            this.spinner.succeed();
+
+            this.spinner.start(`Inspecting state after reconciliation..`);
+            const currentState = await this.terraformShow();
+
+            const { outputs } = currentState.values;
+
+            if ('main_vpc_id' in outputs &&
+                'manager_dns' in outputs &&
+                'manager_ip' in outputs &&
+                'block_storage' in outputs) {
+                if (outputs.main_vpc_id.value.length > 0 &&
+                    outputs.manager_dns.value.length > 0 &&
+                    outputs.manager_ip.value.length > 0 &&
+                    outputs.block_storage.value.length > 0) {
+
+                    this.spinner.succeed();
+                    this.spinner.start(`Looking for the manager EC2 instance..`);
+
+                    await this.checkManagerLivenessWithAWSCLI({
```

Comment:
You can check this right away, we know manager ip already because it comes from `node.json`